pkg/bless/client.go

```diff
@@ -198,7 +198,12 @@ func (c *Client) getCert(ctx context.Context, payload *LambdaPayload) (*LambdaRe
 	if err != nil {
 		return nil, errors.Wrap(err, "Could not serialize lambda payload")
 	}
-	responseBytes, err := c.Aws.Lambda.Execute(ctx, c.conf.LambdaConfig.FunctionName, payloadB)
+	var responseBytes []byte
+	if c.conf.LambdaConfig.FunctionVersion != nil {
+		responseBytes, err = c.Aws.Lambda.ExecuteWithQualifier(ctx, c.conf.LambdaConfig.FunctionName, *c.conf.LambdaConfig.FunctionVersion, payloadB)
```

Comment:
I think aws.Lambda already deals with optional qualifiers, what do you think about making that an optional param into ExecuteWithQualifier and letting AWS deal with that conditional? Would involve merging https://github.com/chanzuckerberg/go-misc/pull/22

Reply:
That way we get rid of this conditional

Reply:
Yeah! That's what I was trying to achieve but I'm just being noob at go :) I will wait until you merge that, and then re-run `dep ensure -update`

Reply:
done!

Reply:
doing pkg updates https://github.com/chanzuckerberg/blessclient/pull/81 because somehow i lost the dep version lock file